.github/workflows/release.yml

```diff
@@ -391,6 +391,7 @@ jobs:
         working-directory: ${{env.BIN_DIR}}
         run: |
           ./${{env.TEST_BIN_FILENAME}} \
+            --chain=frequency-bench \
```

Comment:
Make sure that we are using the most recent runtime from the binary to generate the metadata.